common/djangoapps/student/urls.py

```diff
@@ -17,6 +17,7 @@
     path('accounts/disable_account_ajax', views.disable_account_ajax, name="disable_account_ajax"),
     path('accounts/manage_user_standing', views.manage_user_standing, name='manage_user_standing'),
 
+    path('api/change_email_settings', views.change_email_settings, name='change_email_settings_api'),
```

Comment:
why we need to create this alias?

Reply:
From the code If I think as a layman that's its is quite confusing and nobody could say that it will be using for both web view and API simultaneously without exploring the codebase.

That's why i decided to add it that way
Let me know if does that make sense or should I remove it either.

Reply:
I don't think it needs to be a web view. It is being called [here](https://github.com/edx/edx-platform/blob/a8d1fb11ab9806d243ae30e998b07f4b20e788fc/lms/static/js/dashboard/legacy.js#L215) from ajax request.

Reply:
how about if we only use that router for it `api/change_email_settings` and remove the other one?